R/rocco.R

```diff
@@ -26,25 +29,31 @@
 #'   # The below will simply create a static HTML site without opening it.
 #'   rocco("/path/to/package", output_dir = "/my/html/dir", browse = FALSE)
 #' }
-rocco <- function(directory, output_dir = tempdir(), browse = interactive(), gh_pages = FALSE) {
+rocco <- function(directory, output_dir = tempdir(), browse = interactive(),
+  rocco = TRUE, staticdocs = TRUE, gh_pages = FALSE) {
   if (missing(directory)) directory <- "."
   stopifnot(is.character(directory), length(directory) == 1,
             is.character(output_dir), length(output_dir) == 1,
-            isTRUE(browse) || isFALSE(browse),
             is_package_directory(directory))
 
-  tryCatch({
-    rocco_(directory, output_dir)
-    if (isTRUE(gh_pages)) {
-      `commit_to_gh_pages!`(directory, output_dir)
-    }
+  if (isTRUE(staticdocs) && !staticdocs_exist()) { write_staticdocs() }
```

Comment:
I'm confused. Don't we have to pass `directory`? We want static docs for an arbitrary package, not for rocco itself.